app/api/helpers/errors.py

```diff
@@ -14,8 +14,8 @@ class ErrorResponse:
     headers = {'Content-Type': 'application/vnd.api+json'}
 
     def __init__(self, source, detail, title=None, status=None):
-        """Initialize a jsonapi ErrorResponse Object
-
+        """
+        Initialize a jsonapi ErrorResponse Object
```

Comment:
Unneeded change, there should be no changes in this file

Reply:
@iamareebjamal Will revert it back to the original file